packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -64,3 +65,9 @@ export function getSlugOrRequestedSlug(slug: string) {
     ],
   } satisfies Prisma.TeamWhereInput;
 }
+
+export function getUserOrgDomain(organization: RouterOutputs["viewer"]["me"]["organization"]): string {
+  const orgSlug = organization.slug ?? organization.metadata?.requestedSlug;
```

Comment:
@hariombalhara would this satisfy this comment?
https://github.com/calcom/cal.com/pull/10700#discussion_r1299798360